src/main/java/com/icthh/xm/ms/dashboard/web/rest/DefaultProfileResource.java

```diff
@@ -119,6 +123,7 @@ public DefaultProfileResource(DefaultProfileService defaultProfileService,
     @DeleteMapping("/default-profiles/{id}")
     @Timed
     @PreAuthorize("hasPermission({'id': #id}, 'defaultProfile', 'DEFAULT_PROFILE.DELETE')")
+    @PrivilegeDescription("Privilege to delete the defaultProfile by id")
```

Comment:
replace defaultProfiles to more informative message. Describe what is defaultProfiles.